install/operator/deploy/service_account.yaml

```
@@ -2,3 +2,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: syndesis-operator
+  labels:
+    app: syndesis
+    syndesis.io/app: syndesis
+    syndesis.io/type: operator
```

Perhaps like this, to distinguish from the operator? Not sure what labels we set on other service accounts...

```suggestion
    syndesis.io/type: service-account
```

---

It seems to refers to the components the system account is managing, for instance:
https://github.com/syndesisio/syndesis/blob/e14633aa7e591f635a9466ba4e28e6b5fa66557a/install/operator/pkg/generator/assets/infrastructure/04-syndesis-server.yml.mustache#L1-L9

or
https://github.com/syndesisio/syndesis/blob/e14633aa7e591f635a9466ba4e28e6b5fa66557a/install/operator/pkg/generator/assets/infrastructure/02-syndesis-service-accounts.yml.mustache#L1-L9

so I guess operator is acceptable, but not sure whether we have some guidelines for it, this was again a c&p from:

https://github.com/syndesisio/syndesis/blob/e14633aa7e591f635a9466ba4e28e6b5fa66557a/install/operator/pkg/generator/assets/install/operator.yml.tmpl#L9